test/e2e/cluster_extension_install_test.go

```diff
@@ -154,7 +153,7 @@ func TestClusterExtensionInstallReResolvesWhenNewCatalog(t *testing.T) {
 		}
 		assert.Equal(ct, metav1.ConditionFalse, cond.Status)
 		assert.Equal(ct, ocv1alpha1.ReasonResolutionFailed, cond.Reason)
-		assert.Equal(ct, fmt.Sprintf("no package %q found", pkgName), cond.Message)
+		assert.Contains(ct, cond.Message, fmt.Sprintf("no package %q found", pkgName))
```

Comment:
Nit: I think this can be reverted back to `assert.Equal`. `assert.Contains` doesn't seem necessary here.